metricbeat/module/elasticsearch/index_recovery/_meta/fields.yml

```diff
@@ -26,9 +26,9 @@
       description: >
         Target node id.
     - name: target.host
-      type: ip
+      type: keyword
       description: >
-        Target node host address.
+        Target node host address (could be IP address or hostname).
```

Comment:
Hmmm, so in a sense, that's also a bugfix? Hostnames here were expected, but not supported? 👍

Reply:
This is correct. Our discussions in the context of ECS made me look into this field closer and realize that the data type here ought to be `keyword`, not `ip`.